zc_common/remote_resource/mixins.py

```diff
@@ -1,24 +0,0 @@
-"""
```

Comment:
[Proof of it's non-use](https://github.com/search?q=org%3AZeroCater+MultipleIDMixin&type=Code)